docs/api/options.md

```diff
@@ -206,6 +207,32 @@ Component will be attached to DOM when rendered if set to `true`.
 
 Set the component instance's `$attrs` object.
 
+## propsData
+
+- type: `Object`
+
+Set the component instance's props. 
+
+Example:
+
+```js
+const Component = {
+  template: '<div>{{ msg }}</div>',
+  props: ['msg']
+}
+const wrapper = mount(Component, {
+  propsData: {
+    msg: 'aBC'
+  }
+})
+expect(wrapper.text()).toBe('aBC')
+```
+
+::: tip 
+It's worth noting that `propsData` is actually a [Vue API](https://vuejs.org/v2/api/#propsData), not a proprietary
```

Comment:
Great explanation, although can you remove the word proprietary, it sounds too formal. So:

`It's worth noting that `propsData` is actually a [Vue API](https://vuejs.org/v2/api/#propsData), not a `vue-test-utils` option.`